lesson1-oop-1/ItemWeightComparator.ts

```diff
@@ -0,0 +1,11 @@
+import Item from './Item';
+import ItemComparator from './ItemComparator';
+
+export default class ItemWeightComparator implements ItemComparator {
+    public compare(first: Item, second: Item) {
```

Comment:
If the weights are equal, this method should call the compareTo(other: Item) method of the first Item and return the resulting value.